data/successors/successor weapons.txt

```diff
@@ -0,0 +1,389 @@
+
+effect "successor pulse laser impact"
+	sprite "effect/successor pulse laser impact"
+		"frame rate" 30
+		"random start frame"
+	lifetime 3
+	"velocity scale" 0
+
+outfit "Overcharged Pulse Laser"
+	category "Guns"
+	series "Guns: Successors"
+	index 40
+	cost 84000
+	thumbnail "outfit/overcharged pulse laser"
+	licenses
+		"Successor"
+	"mass" 12
+	"outfit space" -12
+	"weapon capacity" -12
+	"gun ports" -1
+	weapon
+		sprite "projectile/successor pulse laser"
+			"start frame" 2
+			"frame rate" 8
+			"rewind"
+		sound "overcharged pulse laser"
+		"fire effect" "moonbeam fleck" 3
+		"hit effect" "overcharged pulse laser impact"
+		"inaccuracy" 5.5
+		"velocity" 490
+		"lifetime" 1
+		"burst count" 4
+		"burst reload" 1
+		"reload" 7
+		"firing energy" 9.1
+		"firing heat" 30.8
+		"shield damage" 14
+		"hull damage" 13.3
+		"piercing" 0.05
+		"cluster"
+	description `Unable to access large quantities of the high-quality weaponry produced by the High Houses, the People's Houses have had to work with what is available to them: mostly the relatively low-powered pulse lasers produced independently in Successor space. This pulse laser has had its safeties bypassed and its optics upgraded, allowing it to do more damage to shields at a lower power cost in exchange for producing substantially more heat.`
+
+outfit "Overcharged Laser Turret"
+	category "Turrets"
+	series "Turrets: Successors"
+	index 30
+	cost 136000
+	thumbnail "outfit/overcharged pulse laser turret"
+	licenses
+		"Successor"
+	"mass" 24
+	"outfit space" -24
+	"weapon capacity" -24
+	"turret mounts" -1
+	weapon
+		sprite "projectile/successor pulse laser"
+			"start frame" 2
+			"frame rate" 8
+			"rewind"
+		"hardpoint sprite" "hardpoint/overcharged pulse laser turret"
+		"hardpoint offset" 8
+		sound "overcharged pulse laser"
+		"fire effect" "moonbeam fleck" 3
+		"hit effect" "overcharged pulse laser impact"
+		"inaccuracy" 5.5
+		"velocity" 490
+		"lifetime" 1
+		"turret turn" 3.5
+		"burst count" 4
+		"burst reload" 1
+		"reload" 7
+		"firing energy" 12.6
+		"firing heat" 43.05
+		"shield damage" 19.6
+		"hull damage" 18.667
+		"piercing" 0.05
+		"cluster"
+	description `Hand-crafted by the more colorful members of the People's Houses, this bespoke overcharged Pulse Laser turret has had even more power pumped into its laser generator compared to the fixed version, producing a truly prodigious amount of heat for a weapon of its size. Despite its inelegant design, it is nonetheless a fearsome weapon, especially considering that it retains the ability to partially punch through shielding.`
+
+effect "overcharged pulse laser impact"
+	sprite "effect/overcharged pulse laser impact"
+		"frame rate" 30
+		"random start frame"
+	lifetime 6
+	"velocity scale" 0
+
+outfit `"Sjeja" Kinetic Lance`
```

Comment:
This is both nontrivial to support energy-wise, requiring nearly the full output of a Yellow Sun or Epoch Cell, and ammo-limited. In exchange for that, it can deal major damage, with almost no reaction time, at very long range, and with piercing and penetration.
In the player's hands, I think it's balanced out by the limitation of requiring ammunition. Its effectiveness is reasonably in-line with some other high-tier missile weapons - its DPS is slightly below that of the T2 Firestorm, and considerably below that of the T2.5 Finisher Pod, while the weapon itself is larger and hugely more power-hungry than either. Getting to and from Successor space is nontrivial too, so sourcing ammunition is more difficult than for many other missiles.

My main concern is the lack of options for the player if they end up on the recieving end of the Sjeja in future story content. Since it reaches its full range in only 8 frames, there's not really scope for dodging it, and at that kind of long range circling round faster than the opponent can turn isn't really an option either (even if the firing ship has an extremely low turn rate of only 40 degrees/sec, at 1900 range you'd need a speed of over 1200 to stay ahead). The high hit force and firing force make closing the distance difficult too, particularly in lighter ships that are pushed more.